pkg/netpol/connlist/connlist.go

```diff
@@ -632,6 +660,30 @@ func (ca *ConnlistAnalyzer) getConnectionsList(pe *eval.PolicyEngine, ia *ingres
 	return connsRes, peers, nil
 }
 
+// checkFocusWorkloadsExistence checks if the peers in the given focus-workloads (or focus workload-peers) list exist;
```

Comment:
is the input arg `focusWlsList` expected to be a union of them?
if so specify that in comment.

Reply:
no, it gets one of them only